lib/shopify-cli/commands.rb

```diff
@@ -6,10 +6,16 @@ module Commands
       default: 'help',
       contextual_resolver: nil,
     )
+    @core_commands = []
 
     def self.register(const, cmd, path = nil)
       autoload(const, path) if path
       Registry.add(->() { const_get(const) }, cmd)
+      @core_commands.push(cmd)
+    end
+
+    def self.core_command?(cmd)
+      @core_commands.include?(cmd)
```

Comment:
Do we need to consider case sensitivity?

Reply:
We don't modify the commands to be `downcase` (which is probably preferable) when we register them, so the registrations would also need to be uniformly case'd in order for the check to be case sensitive.  But maybe we should ... @tanema?

Reply:
@mkevinosullivan maybe open an issue to track this. no one should be making uppercase commands but we should also make sure of that ourselves.